paper/paper.tex

```diff
@@ -0,0 +1,149 @@
+\label{fig:arch_diagram}
+\end{figure}
+
+\subsection{Split Value Representations}
+Each vote in this system must be represented as an integer $x, 0 \leq x < M$, where $M$ is some integer chosen so there are enough values to represent all possible vote choices. The split value concept used is a form of secret sharing among all the Mix Servers in the Proof Server. A split value representation of a value $x$ is a vector
+\begin{equation} \label{eq:svr}
+    X = (u, v)
+\end{equation}
+such that
+\begin{equation}
+    x = \mathit{VAL}(X) = (u + v) \mbox{ mod } M
+\end{equation}
+
+This idea is then extended so that $x$ can be represented with a n-tuple of such vectors. The Proof Server has a configurable number of rows allowed, this row size determines the number of ways each vote is split again. In our example network in Fig. \ref{fig:arch_diagram}, the row size is $3$ so now we would represent the vote $x$ as the vector $(X, Y, Z)$
+\begin{equation} \label{eq:comt}
+    x = \mathit{VAL}(X) + \mathit{VAL}(Y) + \mathit{VAL}(Z) (\mbox{mod } M)
+\end{equation}
+
+The use of an n-tuple to represent the vote $x$ allows the Mix Servers to share the secret among them, performing the mixing operations without any one server knowing the details of an individual vote. That is, row 1 in the Proof Server only ever operates on $(X_1, X_2, \dots, X_n)$ for $n$ votes.
+
+Furthermore, the split value representation using $(u, v)$ for each component of the vote $x$ in Equation \ref{eq:comt} allows for a convenient property where the Proof Server can prove the equivalence of two vote values without revealing the values in their entirety. Only part of the value is needed ($u$ or $v$, but not both) in addition to a value $t$. For example, consider the values $x$ and $y$
+\begin{align*} \label{eq:comt}
+    x = (u_1 + v_1) \mbox{ mod } M \\
+    y = (u_2 + v_2) \mbox{ mod } M \\
+    t_u = (u_2 - u_1) \mbox{ mod } M \\
+    t_v = (v_1 + v_2) \mbox{ mod } M
+\end{align*}
+If one is given $(t_u, u_1, u_2)$ or $(t_v, v_1, v_2)$, either is sufficient to validate if $x == y$, but neither triple reveals the true values of $x$ or $y$. This concept is what allows the Proof Server to prove to external parties that the mixed output matches the original cast votes, without revealing what those vote values are. This is important because it maintains privacy for voters and avoids the possibility of vote coercion because voters cannot prove their exact vote value.
+
+Further details on the specifics of the protocol are outside the scope of this report.
+
+\section{Implementation}
```

Comment:
Maybe also worth adding the simplification of secure communication between tablets and PS